features/appmgt/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/mobileapp/partials/add-mobileapp-version.hbs

```diff
@@ -40,19 +40,18 @@
         <form class="form-horizontal" id='form-asset-create' action="/publisher/api/asset/mobileapp" method="post" enctype="multipart/form-data">
 
 
-            <input type="hidden" id="appmeta" name="appmeta" value="">
+            <input type="hidden" id="appmeta" name="appmeta" value=""/>
 
-            <input type="hidden" name="version" id="version" value="1.0">
-            <input type="hidden" name="provider" value="1WSO2Mobile">
+            <input type="hidden" name="version" id="version" value="1.0"/>
```

Comment:
Not sure, I think it is used as a default value if something goes wrong when getting the user input. I merely added the closing "/" to this line (45).